dev-python/ipywidgets/ipywidgets-8.1.2-r1.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( pypy3 python3_{10..12} )
+PYTHON_REQ_USE="threads(+)"
+
+inherit distutils-r1 pypi
+
+DESCRIPTION="IPython HTML widgets for Jupyter"
+HOMEPAGE="
+	https://ipywidgets.readthedocs.io/
+	https://github.com/jupyter-widgets/ipywidgets/
+	https://pypi.org/project/ipywidgets/
+"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="amd64 arm arm64 ~loong ppc64 ~riscv x86 ~amd64-linux ~x86-linux"
+
+RDEPEND="
+	>=dev-python/comm-0.1.3[${PYTHON_USEDEP}]
+	>=dev-python/ipython_genutils-0.2.0[${PYTHON_USEDEP}]
+	>=dev-python/traitlets-4.3.1[${PYTHON_USEDEP}]
+	>=dev-python/widgetsnbextension-4.0.10[${PYTHON_USEDEP}]
+	>=dev-python/jupyterlab_widgets-3.0.10[${PYTHON_USEDEP}]
```

Comment:
Shouldn't this be an optional dependency?

Reply:
Do you mean that on dev-python/jupyterlab_widgets? It is the same as for dev-python/widgetsnbextension which is the legacy way, latest jupyter requires dev-python/jupyterlab_widgets, see the [docs](https://ipywidgets.readthedocs.io/en/latest/user_install.html). Right now ipywidgets disables jupyterlab_widgets because the package is not in portage, the result is that the widgets are not available in the browser